### PR TITLE
Add/Edit a few Basic Machine pages

### PR DESCRIPTION
**Pull request by svr333**
Do mention any mistakes.

## New Pages
- Automated Panning Machine
- Grind Stone
- Ore Crusher
- Compressor
- Pressure Chamber

## Edited pages
- Magic workbench
- Armor Forge

### CODE REVIEW

**Comment by TheBusyBiscuit** [comment]
I think you shouldn't have replaced the old magic workbench image.

**Comment by TheBusyBiscuit** [comment]
Nor the Ore Washer...

File: pages/Magic-Workbench.md [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
+A complete setup should look like this:
+
+![Magic Workbench Setup](https://raw.githubusercontent.com/TheBusyBiscuit/Slimefun4-Wiki/images/multiblock-magic-workbench.png)
+
+You can also place an [Output Chest](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Output-Chest) next to the dispenser to collect your items. If there is no [Output Chest](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Output-Chest), then it will be put in the dispenser.
 
 ## Usage
-Place the items in the correct pattern in the Dispenser, then right-click the Crafting Table. If the recipe is correct, your item will appear in the Dispenser.<br>
-An [Output Chest](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Output-Chest) can be placed next to the Dispenser to collect your crafted items.
+It is used to craft many [Magical Items](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Magical-Items) and [Magical Gadgets](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Magical-Gadgets).
```
Comment (TheBusyBiscuit) [inline pages/Magic-Workbench.md]: I think you shouldn't remove the message about placing the items into the dispenser.

File: pages/Ore-Crusher.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+The Ore Crusher is a multiblock structure, it consists of a dispenser, a nether brick fence and 2 iron bars.<br>
+
+A complete setup should look like this:
+
+![Ore Crusher Setup](https://raw.githubusercontent.com/TheBusyBiscuit/Slimefun4-Wiki/master/images/multiblock-ore-crusher.png)
+
+You can also place an [Output Chest](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Output-Chest) next to the dispenser to collect your resources. If there is no [Output Chest](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Output-Chest), then it will be put in the dispenser.
+
+## Usage
+The Ore Crusher is mainly used to crush ores into their minerals, but it has other uses too.
```
Comment (TheBusyBiscuit) [inline pages/Ore-Crusher.md]: "Check out the Ore Crusher in your Slimefun Guide for more"

File: pages/Pressure-Chamber.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+The Pressure Chamber is a multiblock structure, it consists of a dispenser, 2 smooth stone slabs, a piece of glass, a cauldron and 4 pistons.<br>
+
+A complete setup should look like this:
+
+![Pressure Chamber Setup](https://raw.githubusercontent.com/TheBusyBiscuit/Slimefun4-Wiki/master/images/multiblock-pressure-chamber.png)
+
+You can also place an [Output Chest](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Output-Chest) next to the dispenser to collect your items. If there is no [Output Chest](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Output-Chest), then it will be put in the dispenser.
+
+## Usage
+The Pressure Chamber is used to put pressure on items.
```
Comment (TheBusyBiscuit) [inline pages/Pressure-Chamber.md]: This is a wiki, please be more precise than that.
State that it is used to craft various items and that they should look into the Slimefun Guide for more.

File: pages/Grind-Stone.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+The Grind Stone is a multiblock structure, it consists of a dispenser and a fence.<br>
+
+A complete setup should look like this:
+
+![Grind Stone Setup](https://raw.githubusercontent.com/TheBusyBiscuit/Slimefun4-Wiki/master/images/multiblock-grind-stone.png)
+
+You can also place an [Output Chest](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Output-Chest) next to the dispenser to collect your resources. If there is no [Output Chest](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Output-Chest), then it will be put in the dispenser.
+
+## Usage
+The Grind Stone can be used to grind items/blocks into other items/blocks.
```
Comment (TheBusyBiscuit) [inline pages/Grind-Stone.md]: This is a wiki, please be more precise than that.
State that it is used to craft various items and that they should look into the Slimefun Guide for more.

File: pages/Compressor.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+The Compressor is a multiblock structure, it consists of a dispenser, a nether brick fence and 2 pistons.<br>
+
+A complete setup should look like this:
+
+![Compressor Setup](https://raw.githubusercontent.com/TheBusyBiscuit/Slimefun4-Wiki/master/images/multiblock-compressor.png)
+
+You can also place an [Output Chest](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Output-Chest) next to the dispenser to collect your compressed items. If there is no [Output Chest](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Output-Chest), then it will be put in the dispenser.
+
+## Usage
+The Compressor is used to compress items.
```
Comment (TheBusyBiscuit) [inline pages/Compressor.md]: This is a wiki, please be more precise than that.
State that it is used to craft various items and that they should look into the Slimefun Guide for more.

**Comment by svr333** [comment]
> 
> 
> I think you shouldn't have replaced the old magic workbench image.

The thing with the magic workbench was that it wouldn't load on the page, so I reworked the entire page.

File: pages/Automated-Panning-Machine.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+It's an automated version of the [Gold Pan](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Gold-Pan) and the [Nether Gold Pan](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Nether-Gold-Pan).
+
+## Obtaining
+The Automated Panning Machine is a multiblock structure, it consists of a cauldron and a trapdoor.<br>
+
+A complete setup should look like this:
+
+![Automated Panning Machine Setup](https://raw.githubusercontent.com/TheBusyBiscuit/Slimefun4-Wiki/master/images/multiblock-automated-panning-machine.png)
+
+You can also place an [Output Chest](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Output-Chest) next to the dispenser to collect your items. If there is no [Output Chest](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Output-Chest), then it will be put in the dispenser.
```
Comment (TheBusyBiscuit) [inline pages/Automated-Panning-Machine.md]: There is no dispenser in this structure...

**Comment by svr333** [comment]
After some struggling with the bloody image I resolved it, anyway, do I re-implement the old image of the Magic Workbench? I removed it since it wouldn't show up on anyone's (3 people) their github, it would fail to load.

File: pages/Automated-Panning-Machine.md [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+
+A complete setup should look like this:
+
+![Automated Panning Machine Setup](https://raw.githubusercontent.com/TheBusyBiscuit/Slimefun4-Wiki/master/images/multiblock-automated-panning-machine.png)
+
+You can also place an [Output Chest](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Output-Chest) next to the cauldron to collect your items. If there is no [Output Chest](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Output-Chest), then it will be thrown on top of the trapdoor.
+
+## Usage
+It's a faster version of the [Gold Pan](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Gold-Pan) and the [Nether Gold Pan](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Nether-Gold-Pan) and has the exact same usages as them combined.<br>
+To use the Automated Panning Machine, right click the cauldron with any block that is convertable, it will then proceed to output its converted items after a small animation.
```
Comment (TheBusyBiscuit) [inline pages/Automated-Panning-Machine.md]: You right click the trap door, not the cauldron.

File: pages/Automated-Panning-Machine.md [hunk 8]
```diff
@@ -0,0 +1,15 @@
+The Automated Panning Machine is in the [Basic Machines](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Basic-Machines) category. It is a multiblock structure.<br>
+It's an automated version of the [Gold Pan](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Gold-Pan) and the [Nether Gold Pan](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Nether-Gold-Pan).
+
+## Obtaining
+The Automated Panning Machine is a multiblock structure, it consists of a cauldron and a trapdoor.<br>
```
Comment (TheBusyBiscuit) [inline pages/Automated-Panning-Machine.md]: You may also want to mention that ANY wooden trapdoor works.

File: pages/Magic-Workbench.md [hunk 9]
```diff
@@ -1,12 +1,15 @@
...
 
 ## Obtaining
-The Magic Workbench is a Multiblock structure which consists of a Bookshelf, Crafting Table and Dispenser placed side by side.<br>
-A complete Magic Workbench looks like this :
+The Magic Workbench is a multiblock structure which consists of a bookshelf, crafting table and dispenser placed side by side.<br>
 
-![Magic Workbench setup](https://raw.githubusercontent.com/TheBusyBiscuit/Slimefun4-Wiki/images/multiblock-magic-workbench.png)
+A complete setup should look like this:
+
+![Magic Workbench Setup](https://raw.githubusercontent.com/TheBusyBiscuit/Slimefun4-Wiki/images/multiblock-magic-workbench.png)
```
Comment (TheBusyBiscuit) [inline pages/Magic-Workbench.md]: There is a 'master' missing in the path (right between the repository name and 'images').
That is probably also why the image isn't showing.

Als, please revert to the previous magic workbench image, I think it's more cleaner than your blue background one.

File: pages/Pressure-Chamber.md [hunk 10]
```diff
@@ -0,0 +1,13 @@
...
+The Pressure Chamber is a multiblock structure, it consists of a dispenser, 2 smooth stone slabs, a piece of glass, a cauldron and 4 pistons.<br>
+
+A complete setup should look like this:
+
+![Pressure Chamber Setup](https://raw.githubusercontent.com/TheBusyBiscuit/Slimefun4-Wiki/master/images/multiblock-pressure-chamber.png)
+
+You can also place an [Output Chest](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Output-Chest) next to the dispenser to collect your items. If there is no [Output Chest](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Output-Chest), then it will be put in the dispenser.
+
+## Usage
+The Pressure Chamber is used to put pressure on items, check out the Pressure Chamber in the Slimefun Guide in-game for more recipes.
```
Comment (TheBusyBiscuit) [inline pages/Pressure-Chamber.md]: You may want to add that you have to right-click the cauldron.

File: pages/Pressure-Chamber.md [hunk 11]
```diff
@@ -0,0 +1,14 @@
...
+
+A complete setup should look like this:
+
+![Pressure Chamber Setup](https://raw.githubusercontent.com/TheBusyBiscuit/Slimefun4-Wiki/master/images/multiblock-pressure-chamber.png)
+
+You can also place an [Output Chest](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Output-Chest) next to the dispenser to collect your items. If there is no [Output Chest](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Output-Chest), then it will be put in the dispenser.
+
+## Usage
+The Pressure Chamber is used to put pressure on items, check out the Pressure Chamber in the Slimefun Guide in-game for more recipes.<br>
+To use the Pressure Chamber, put the items (with the correct pattern) in the dispenser and when correct your items will be output when you right click the cauldron.<br>
```
Comment (TheBusyBiscuit) [inline pages/Pressure-Chamber.md]: I would remove the 'when correct'

**Pull request merged**